README.md

```diff
@@ -103,11 +125,10 @@ To learn more about the platform, check out our [documentation](https://github.c
 - Native installation
 
   ```sh
-  # launch semantic network processing machine
-  ./scripts/run_sc_machine.sh
-  # *in another terminal*
-  # launch semantic interfaces interpreter at localhost:8000
-  ./scripts/run_sc_web.sh
+  # to run sc-machine, see https://ostis-ai.github.io/sc-machine/build/quick_start/#run-sc-machine-in-debug
+  ./sc-machine/build/Debug/bin/sc-builder -i repo.path -o kb.bin --clear
```

Comment:
error like in docker
![image](https://github.com/user-attachments/assets/9e0f5701-6e42-43de-a3aa-ec5ce09c015f)

Reply:
ok, I changed branch in manager and it was fixed